src/Models/Traits/HasAttributeOptions.php

```diff
@@ -8,18 +8,29 @@
 trait HasAttributeOptions
 {
     protected static $hasAttributeOptions = true;
+    protected static $attributeCache = [];
 
     protected function options(): Attribute
     {
-        // Sort by store_id first to always get the higher store id if there are two.
-        return Attribute::get(fn () => $this->attributeOptions
-            ->sortBy('store_id')
-            ->keyBy('option_id')
-        );
+        return Attribute::get(function() {
```

Comment:
Object caching is there by default? https://laravel.com/docs/master/eloquent-mutators#value-object-caching, but there is also a caching option: https://laravel.com/docs/master/eloquent-mutators#accessor-caching